src/Console/Command/Compile.php

```diff
@@ -545,6 +547,10 @@ private function configureMetadata(Configuration $config, Box $box, CompilerLogg
                 'Setting metadata'
             );
 
+            if (is_file($metadata) && is_readable($metadata)) {
```

Comment:
I wonder if it'd also be possible to confirm if this file is valid PHP code (maybe invoke `php -l` here as well?)

Reply:
Sorry, but I think here it's too much, especially because `$metadata = require $metadata;` allow to be alert if the file included as parse error

Reply:
What version of the PHP interpreter are you using?

On my workstation, I am running PHP 7.4.20 and I don't have parse errors or warnings returned, which is why I suggested to confirm if the file is valid PHP.

I tested using the following file `test.php`:
```php
<?php

$file = $argv[1];
$result = require $file;

var_dump($result);  #
```

with
1. an empty file named `emptyfile` (created by running `touch emptyfile`) and
2. pythonfile.py with the contents:
```python
print("Hello world!")
```

Running both of these scenarios ran without error, with the contents of the file being emitted to stdout, which aligns with the PHP documentation of [`include`](https://www.php.net/manual/en/function.include.php), specifically this section:
>When a file is included, parsing drops out of PHP mode and into HTML mode at the beginning of the target file, and resumes again at the end. For this reason, any code inside the target file which should be executed as PHP code must be enclosed within valid PHP start and end tags.

Let's defer this decision (on whether PHP syntax checking should be added for a better Developer Experience) to @box-project / @humbug